tests/services/inventory/test_data/mock_gcp_results.py

```diff
@@ -428,6 +428,26 @@
 }}
 """
 
+CRM_PROJECT_IAM_POLICY_MEMBER_MULTI_ROLES = """
+{
+ "version": 1,
+ "bindings": [
+  {
+   "role": "roles/appengine.codeViewer",
```

Comment:
Actually, I like the previous test data better.  Where there is another role, that does not contain the abc_user.  Can you please add that, just to make sure this test is robust?